src/main/java/org/frc5687/powerup/robot/Constants.java

```diff
@@ -151,6 +151,11 @@
         public static final double ENCODER_TOP = 340;
 
         public static final double HOLD_SPEED = 0.1;
+
+        public class Pot {
+            public static final double TOP = 391.0;
+            public static final double BOTTOM = 265.0;
+        }
```

Comment:
I'm concerned that these pot values are so close together.  Do we have another pot we can use with a larger range?